### DISCUSSION TITLE
Docker Host and Socket Path to be configured via Env variables - UnixSocketStrategy

### DISCUSSION THREAD

**Discussion by DineshKuppan** [testcontainers/testcontainers-node#371] [General]
This request is for Mac OSX M1 Pro - Colima container  - Due to wait strategy of UnixSocketStrategy for docker container issue (#362)

testcontainers-node - v8.6.1 (Is it possible to create a new version(v8.6.2) from this release tag to address for workaround mentioned in the above issue)

Node JS Version - v16.15.0
colima - 0.4.2

Note: This can be raised to new release version as well, at least will have the control over via Environment variables

**Environment Variables :**
DOCKER_HOST & DOCKER_SOCKET_PATH

Please check this commit for the changes - [https://github.com/testcontainers/testcontainers-node/compare/master...DineshKuppan:update-docker-host-and-socket-path-env-variables](https://github.com/testcontainers/testcontainers-node/compare/master...DineshKuppan:update-docker-host-and-socket-path-env-variables)

**Comment by cristianrgreco** [comment]
`DOCKER_HOST` and `TESTCONTAINERS_DOCKER_SOCKET_OVERRIDE` are already supported as environment variables. Note that these are used in the `ConfigurationWaitStrategy`, not the `UnixSocketStrategy` (the `DOCKER_HOST` could for example be remote).
